packages/module/patternfly-docs/content/extensions/component-groups/about-component-groups.md

```diff
@@ -0,0 +1,9 @@
+---
+section: extensions
+subsection: Component groups
+id: About component groups
```

Comment:
Probably should say About to match the other nodes on the site.

Reply:
I actually have an issue to update that naming convention across the site because having this many pages simply called "about" has some accessibility implications. So component groups will just be ahead of the curve

Reply:
Sounds good